Makefile

```diff
@@ -1,7 +1,17 @@
+export LIBRARY_TEST:=TRUE  #Used by dp-cookies/cookies/cookies.go L37 to identify whether its running a test.
+
 .PHONY: debug
 debug:
 	go run main.go
 
 .PHONY: test
 test: 
-	go test -race -cover ./...
\ No newline at end of file
+	go test -race -cover ./...
+
+.PHONY: audit
+audit:
+	go list -json -m all | nancy sleuth --exclude-vulnerability-file ./.nancy-ignore
+	
+.PHONY: build	
+build:
+	go build ./...
```

Comment:
I _think_ you have to either build all packages (`./...`) into a folder (`-o [folder]) 

OR

do `go build` only

Reply:
All I'm wanting to do is to make sure all go files build within the repo, which `./...` seems to be designed to do e.g. https://stackoverflow.com/a/42400924

Some libraries might produce a binary, but I think most won't.  As I'm not looking to create a binary/executable, I don't know if the `-o` is applicable, as that is used to rename a binary I think?  Or am I missing something?